front/app/components/Form/Components/Fields/index.tsx

```diff
@@ -71,7 +71,7 @@ const Fields = ({
     (
       error: ErrorObject,
       _translate: Translator,
-      uischema?: UISchemaElement
+      uischema?: UISchemaElement & { label?: string }
```

Comment:
@IvaKop This doesn't feel super clear to me when reading the code - do you think we could add a comment or perhaps make this a new singular Type?

Reply:
Only comment I had - code looks good to me otherwise!! Doing a functional review now :)

Reply:
Turns out we already had this type defined elsewhere so I was able to reuse it. Nice catch!